### PR TITLE
add default seccomp profile

### PR DESCRIPTION
**Pull request by psychosis448**
# Changes

Add default seccomp profile to PodSecurityPolicy.  
Otherwise pods can not currently start due to missing default seccomp profile

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [ ] Release notes block below has been filled in or deleted (only if no user facing changes)

### CODE REVIEW

**Review by wlynch — APPROVED** [review]
/lgtm

File: config/101-podsecuritypolicy.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ metadata:
     app.kubernetes.io/part-of: tekton-pipelines
   annotations:
     seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: runtime/default
```
Comment (vdemeester) [inline config/101-podsecuritypolicy.yaml:24]: Does it need to be quoted ? (maybe just for the sake of consistency with the other annotations)

Comment (psychosis448) [reply config/101-podsecuritypolicy.yaml:24]: For consistency, yes, it should be quoted.
Sorry, I missed that.

**Review by vdemeester — APPROVED** [review]

**Comment by jerop** [comment]
known sidecar test flake: https://github.com/tektoncd/pipeline/issues/2656

/test pull-tekton-pipeline-integration-tests

/lgtm

thanks @psychosis448!

**Pull request merged**